amazon-lambda/1.0/modules/ROOT/pages/amazon-lambda-connector-examples.adoc

```diff
@@ -0,0 +1,88 @@
+
+The function name is supplied from a static configuration file called `environment.properties`.
+
+== XML for the Invoke Example
+
+Paste this code into the Studio XML editor to quickly load the flow for this example into your Mule app:
+
+[source,xml,linenums]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns:amazon-lambda="http://www.mulesoft.org/schema/mule/amazon-lambda" xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns="http://www.mulesoft.org/schema/mule/core"
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+http://www.mulesoft.org/schema/mule/amazon-lambda http://www.mulesoft.org/schema/mule/amazon-lambda/current/mule-amazon-lambda.xsd">
+	<flow name="amazon-lambda-invoke-example">
+		<http:listener doc:name="Listener" config-ref="HTTP_Listener_config" path="/invoke"/>
+		<amazon-lambda:create20150331-functions-invocations-by-function-name doc:name="Invoke" config-ref="Test_Aws_Config" functionName="${config.functionName}" xAmzInvocationType="REQUEST_RESPONSE">
+		</amazon-lambda:create20150331-functions-invocations-by-function-name>
+	</flow>
+</mule>
+----
+
+In this example, the payload from the HTTP Listener is passed directly into the body of the *Invoke* operation, which is specified as a `REQUEST_RESPONSE` type, and also tells AWS Lambda that it does not need to store any logs for this operation. This effectively makes this a synchronous call, which waits until the AWS Lambda function is finished processing before it returns its results. 
+
+Alternatively, you can set the *Invocation Type* to `EVENT`, which acts like an asynchronous call and returns an empty response immediately while AWS Lambda performs its action in the background.
+
+== Create an AWS Lambda Function
+
+Depending on your Lambda function, when you run this flow you might need to send in a request body. 
+
+For this example, you can create a Python 3.8 Lambda function like this:
+
+[source,python,linenums]
+----
+def lambda_handler(event, context):
+    message = 'Hello {} {}!'.format(event['first_name'], event['last_name'])
+
+	return {'message' : message}
+----
+
+Consult the official AWS documentation if you don't know how to create a new AWS Lambda function.
+
+This function takes the fields `first_name` and `last_name` from the input in JSON format and sends a greeting using the specified first and last names. 
+
+If you call the http://localhost:8081/invoke endpoint with a JSON body like this:
+
+[source,json]
+----
+{
+    "first_name":"Tyler",
+    "last_name":"Durden"
+}
+----
+
+The Mule flow responds with a code 200 message:
+----
+{
+    "message": "Hello <first_name> <last_name>!"
```

Comment:
@valkyrie69 
you changed the response to `Hello <first_name> <last_name>!`
but the payload is still
```
{
    "first_name":"Tyler",
    "last_name":"Durden"
}
```

shouldn't we also change the payload to 
```
{
    "first_name":"<first_name>",
    "last_name":"<last_name>"
}
```

so the result matches the payload? I think it might end up being a bit confusing if they don't match up. This is also why I originally chose a real example name, so it's clear that A leads to B exactly, which should match the python code.